qemu/tests/block_copy.py

```diff
@@ -497,3 +497,46 @@ def verify_md5s(self):
         """
         file_names = self.params["file_names"]
         map(self.verify_md5, file_names.split())
+
+    def mount_disks_linux(self):
```

Comment:
you may want to use apis provided in this https://github.com/avocado-framework/avocado-vt/pull/1112

Reply:
@hereischen Thanks for review. Yes, `utils_misc.mount()` is introduced in L#513, since I have multiple disk to be mounted, so this function actually is only a loop for mount multi disks(by `utils_misc.mount()`).